docs/components/motor/_index.md

```diff
@@ -459,18 +469,16 @@ Stepper motors will report `true` if they are being powered while holding a posi
 
 **Returns:**
 
-- [(tuple[bool, float])](https://docs.python.org/3/library/functions.html#bool): The bool is `true` if the motor is currently running; `false` if not.
-The float represents the current portion of max power to the motor (between 0 and 1).
+- [(bool)](https://docs.python.org/3/library/functions.html#bool): True if the motor is currently moving; false if not.
 
-For more information, see the [Python SDK Docs](https://python.viam.dev/autoapi/viam/components/motor/index.html#viam.components.motor.Motor.is_powered).
+For more information, see the [Python SDK Docs](https://python.viam.dev/autoapi/viam/components/motor/index.html#viam.components.motor.Motor.is_moving).

```

Comment:
Unsure: should this link be this one instead?

https://python.viam.dev/autoapi/viam/components/motor/motor/index.html#viam.components.motor.motor.Motor.is_moving

Reply:
```suggestion
For more information, see the [Python SDK Docs](https://python.viam.dev/autoapi/viam/components/motor/motor/index.html#viam.components.motor.Motor.is_moving).
```